### DISCUSSION TITLE
Visual Studio 2022 Conflict

### DISCUSSION THREAD

**Discussion by arnicabora** [community/community#12946] [Copilot Conversations] [answered]
Hi, it seems copilot tab shortcut is conflicting with visual studio tab actions like snippets. Is there a way to change copilot shortcuts ?

**Comment by kzu** [comment]
Yeah, integration with code snippets is not great. The `TAB` to accept is not a copilot setting but a built-in VS one, so unfortunately we can't change it. We'll likely disable suggestions while you're expanding a snippet

**Comment by kzu** [comment, marked as answer]
The latest published version (v1.10.0.3 or later) has a fix for the snippets issue! It will now avoid displaying suggestions when you're expanding snippets, and therefore `TAB` will properly accept and move the caret with the snippet as usual. Please give it a try and let us know if it solved the issue for you!

**Reply by arnicabora** [reply]
Sorry for late reply, the snippet problem resolved. Thank you very much !
